### PR TITLE
Add CIP-57 updates

### PR DESCRIPTION
**Pull request by harisang**
This PR updates the text with the changes proposed in CIP-57: https://snapshot.box/#/s:cow.eth/proposal/0x46d4fea1492207cf400fcb7a01141a7d4c730791d658cc77236941fc9eb7dccb

This PR addresses issue #436.

**Note: Should be merged ONLY IF CIP is successful.**

### CODE REVIEW

File: docs/cow-protocol/reference/core/auctions/bonding_pools.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+id: bonding-pools
+---
+
+# Bonding pools
+
+A bonding pool is an Ethereum mainnet safe whose sole owner is the CoW DAO, and which contains $0.5M worth of (yield-bearing) stable coins and 1.5M COW, as specified by [CIP-7](https://snapshot.org/#/cow.eth/proposal/0x267edf7a0bd3c771cfca763322f011ee106d8d5158612c11da29183260d1dba7). When a solver wants to join the solver competition, it first needs to join a bonding pool. The main rationale behind this is that a solver in the live competition gets access to the settlement contract, its buffers and the orderflow of the protocol, which means that the attack vector is non-trivial. For the protocol to be protected from a potentially malicious solver, a substantial bonding pool is required so that in case of misbehavior, there are sufficient funds to reimburse the affected parties (users and/or the DAO).
+
```
Comment (fleupold) [inline docs/cow-protocol/reference/core/auctions/bonding_pools.md:9]: Can we have explicit subsections on
1. How to create a bonding pool (with example transactions)
2. How to join a bonding pool (the vouching part)
3. How to dissolve a bonding pool (the whole project blanc spiel)

File: docs/cow-protocol/reference/core/auctions/rewards.md [hunk 2]
```diff
@@ -75,8 +77,13 @@ The presence of the cap on rewards $$c_u$$, however, makes the problem more comp
...
+- Ethereum mainnet: $$\min\{0.0006 ~\textrm{ETH}, 6 ~\textrm{COW}\}$$,
+- Arbitrum: $$\min\{0.0002 ~\textrm{ETH}, 6 ~\textrm{COW}\}$$,
+- Gnosis Chain: $$\min\{0.15 ~\textrm{xDAI}, 6 ~\textrm{COW}\}$$,
+
+where, again, the conversion from ETH and xDAI to COW is done by using an up-to-date price.
```
Comment (fleupold) [inline docs/cow-protocol/reference/core/auctions/rewards.md]: at the time of ... ?

**Review by fleupold — COMMENTED** [review]
Looks good. Not strictly related to the CiP but can we give more color on how to join a bonding pool? Imagine a solver wants to go through the whole process and only has the public docs at their disposal. Do they get all the information they need to complete this step?

File: docs/cow-protocol/reference/core/auctions/rewards.md (hunk 2 above)
Comment (harisang) [reply docs/cow-protocol/reference/core/auctions/rewards.md]: pushed a clarification
https://github.com/cowprotocol/docs/pull/439/commits/329dc1401fbf422c7737e0e858750d4f0df0b896

**Comment by bram-vdberg** [comment]
> **CLA Assistant Lite bot:** Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you all sign our [Contributor License Agreement](https://github.com/cowprotocol/cla/blob/main/CLA.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.
> 
> I have read the CLA Document and I hereby sign the CLA
> 
> **1** out of **2** committers have signed the CLA.✅ @harisang❌ @bram-vdbergYou can retrigger this bot by commenting **recheck** in this Pull Request

I have read the CLA Document and I hereby sign the CLA

File: docs/cow-protocol/reference/core/auctions/bonding_pools.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+- Mainnet: https://etherscan.io/address/0xb422f2520b0b7FD86f7DA61b32Cc631A59ed7E8F
+- Gnosis: https://gnosisscan.io/address/0xAAA4De096D02AE21729aA31D967E148D4e3Ae501
+- Arbitrum: https://arbiscan.io/address/0xAAA4De096D02AE21729aA31D967E148D4e3Ae501
+- Base: https://basescan.org/address/0xAAA4De096D02AE21729aA31D967E148D4e3Ae501
+:::
+
+# Setting up a bonding pool
+The first step for setting up a bonding pool is to deploy a Gnosis safe on Mainnet with only [the CoW DAO safe](https://etherscan.io/address/0xcA771eda0c70aA7d053aB1B25004559B918FE662) as a signer as is described in [CIP-7](https://snapshot.box/#/s:cow.eth/proposal/0x267edf7a0bd3c771cfca763322f011ee106d8d5158612c11da29183260d1dba7).
+
+Once this safe has been confirmed by the CoW DAO team, the safe should be funded with $500.000 USD in stable coins and 1.500.000 COW tokens. After this is done, the bonding pool can be used to vouch for solvers in the solver competition.
```
Comment (fleupold) [inline docs/cow-protocol/reference/core/auctions/bonding_pools.md]: yield bearing

File: docs/cow-protocol/reference/core/auctions/bonding_pools.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+# Setting up a reduced bonding pool
+Solvers that are currently vouched under the CoW Bonding pool may decide to set up a reduced bonding pool according to [CIP-44](https://snapshot.box/#/s:cow.eth/proposal/0x1b6f1171633ec3d20c4370db37074aa1bd830486d4d0d6c26165915cc42d9412).
+
+The first step for setting up a reduced bonding pool is to deploy a Gnosis safe on Mainnet that has only one signer ([the CoW DAO Solver Payouts safe](0xA03be496e67Ec29bC62F01a428683D7F9c204930)). After this is done and has been confirmed by the CoW DAO team, the solver will deposit $50.000 in yield bearing stable coins or ETH and 500.000 COW tokens to the newly created safe.
+
+# How to join a bonding pool
+This is done by vouching for a solver's submission address and rewards address with the bonding pool address. This is done by calling the `Vouch` method on the [VouchRegister contract](https://etherscan.io/address/0xb422f2520b0b7fd86f7da61b32cc631a59ed7e8f) using the address that owns the bonding pool (the address that sent the full initial funding to the bonding pool).
```
Comment (fleupold) [inline docs/cow-protocol/reference/core/auctions/bonding_pools.md]: according to the cip-7 it's actually the creator and not the initial funder

File: docs/cow-protocol/reference/core/auctions/bonding_pools.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+The vouching contract, that is deployed on every chain the protocol operates on, allows the creators of a bonding pool to publicly vouch for a solver address; this is the only certificate that is currently accepted for proving that a solver has joined a bonding pool. Once such vouching has taken place, then the core team is mandated to proceed with the white-listing of the solver.
+
+:::note
+The vouching contract addresses are the following:
+- Mainnet: https://etherscan.io/address/0xb422f2520b0b7FD86f7DA61b32Cc631A59ed7E8F
+- Gnosis: https://gnosisscan.io/address/0xAAA4De096D02AE21729aA31D967E148D4e3Ae501
+- Arbitrum: https://arbiscan.io/address/0xAAA4De096D02AE21729aA31D967E148D4e3Ae501
+- Base: https://basescan.org/address/0xAAA4De096D02AE21729aA31D967E148D4e3Ae501
+:::
```
Comment (fleupold) [inline docs/cow-protocol/reference/core/auctions/bonding_pools.md]: I think we can remove everything above as the part below is more detailed and complete.

File: docs/cow-protocol/reference/core/auctions/bonding_pools.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+# How to dissolve a bonding pool
+In order to dissolve a bonding pool you must first post a CIP to unwind the bonding pool. This involves first writing a post on [the CoW forum](https://forum.cow.fi/). This can be a very short post detailing the safe creation address and which address funded the pools together with the transaction details for dissolving the bonding pool. [Here is an example from the dissolution of the Project Blanc bonding pool.](https://forum.cow.fi/t/cip-54-dissolve-project-blanc-bonding-pool/2645)
+
+After some discussion in the forum, you will need to post a [CIP on snapshot](https://snapshot.box/#/s:cow.eth). This CIP should contain details of the creation of the bond and details of how the bond will be dissolved. Alongside this it should contain a simulation of the transaction to dissolve the bond that would be executed if the CIP passes. [For example, here is the CIP that proposed the dissolution of the Project Blanc bonding pool.](https://snapshot.box/#/s:cow.eth/proposal/0x2638ee59df1f402421fe69abe76cd0154ec32d8b4ad88a136318c6c8c76b210d)
```
Comment (fleupold) [inline docs/cow-protocol/reference/core/auctions/bonding_pools.md]: indicate exact time needed in forum (there is likely no discussion on those CiPs)

File: docs/cow-protocol/reference/core/auctions/bonding_pools.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+
+# How to leave a bonding pool
+Leaving a bonding pool involves the same process as joining a bonding pool, except that the owner of the bonding pool will call the `invalidateVouching` method instead with the solver's submission address and the bonding pool that vouched for the solver as arguments.
+
+# How to dissolve a bonding pool
+In order to dissolve a bonding pool you must first post a CIP to unwind the bonding pool. This involves first writing a post on [the CoW forum](https://forum.cow.fi/). This can be a very short post detailing the safe creation address and which address funded the pools together with the transaction details for dissolving the bonding pool. [Here is an example from the dissolution of the Project Blanc bonding pool.](https://forum.cow.fi/t/cip-54-dissolve-project-blanc-bonding-pool/2645)
```
Comment (fleupold) [inline docs/cow-protocol/reference/core/auctions/bonding_pools.md]: Note that all solvers need to be unvouched before dissolving

**Review by fleupold — APPROVED** [review]
Almost there, mostly nits about the bonding pool section

**Comment by harisang** [comment]
> > **CLA Assistant Lite bot:** Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you all sign our [Contributor License Agreement](https://github.com/cowprotocol/cla/blob/main/CLA.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.
> > I have read the CLA Document and I hereby sign the CLA
> > **1** out of **2** committers have signed the CLA.✅ @harisang❌ @bram-vdbergYou can retrigger this bot by commenting **recheck** in this Pull Request
> 
> I have read the CLA Document and I hereby sign the CLA

Probably you just need to post the msg as a standalone msg and not as a reply to the original error.

File: docs/cow-protocol/reference/core/auctions/bonding_pools.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
+---
+id: bonding-pools
+---
+
+# Setting up a bonding pool
+The first step for setting up a bonding pool is to deploy a Gnosis safe on Mainnet with only [the CoW DAO safe](https://etherscan.io/address/0xcA771eda0c70aA7d053aB1B25004559B918FE662) as a signer as is described in [CIP-7](https://snapshot.box/#/s:cow.eth/proposal/0x267edf7a0bd3c771cfca763322f011ee106d8d5158612c11da29183260d1dba7).
+
+Once this safe has been confirmed by the CoW DAO team, the safe should be funded with $500.000 USD in yield bearing stable coins and 1.500.000 COW tokens. After this is done, the bonding pool can be used to vouch for solvers in the solver competition.
```
Comment (harisang) [inline docs/cow-protocol/reference/core/auctions/bonding_pools.md]: We should NOT use dots here....

500.000 is five hundred usd with an excess decimal for most of the world

File: docs/cow-protocol/reference/core/auctions/bonding_pools.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+
+# Setting up a reduced bonding pool
+Solvers that are currently vouched under the CoW Bonding pool may decide to set up a reduced bonding pool according to [CIP-44](https://snapshot.box/#/s:cow.eth/proposal/0x1b6f1171633ec3d20c4370db37074aa1bd830486d4d0d6c26165915cc42d9412).
+
+The first step for setting up a reduced bonding pool is to deploy a Gnosis safe on Mainnet that has only one signer ([the CoW DAO Solver Payouts safe](0xA03be496e67Ec29bC62F01a428683D7F9c204930)). After this is done and has been confirmed by the CoW DAO team, the solver will deposit $50.000 in yield bearing stable coins or ETH and 500.000 COW tokens to the newly created safe.
```
Comment (harisang) [inline docs/cow-protocol/reference/core/auctions/bonding_pools.md]: same here with the numbers

File: docs/cow-protocol/reference/core/auctions/bonding_pools.md [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+id: bonding-pools
+---
+
+# Setting up a bonding pool
+The first step for setting up a bonding pool is to deploy a Gnosis safe on Mainnet with only [the CoW DAO safe](https://etherscan.io/address/0xcA771eda0c70aA7d053aB1B25004559B918FE662) as a signer as is described in [CIP-7](https://snapshot.box/#/s:cow.eth/proposal/0x267edf7a0bd3c771cfca763322f011ee106d8d5158612c11da29183260d1dba7).
+
+Once this safe has been confirmed by the CoW DAO team, the safe should be funded with $500.000 USD in yield bearing stable coins and 1.500.000 COW tokens. After this is done, the bonding pool can be used to vouch for solvers in the solver competition.
+
+# Setting up a reduced bonding pool
+Solvers that are currently vouched under the CoW Bonding pool may decide to set up a reduced bonding pool according to [CIP-44](https://snapshot.box/#/s:cow.eth/proposal/0x1b6f1171633ec3d20c4370db37074aa1bd830486d4d0d6c26165915cc42d9412).
```
Comment (harisang) [inline docs/cow-protocol/reference/core/auctions/bonding_pools.md]:
```suggestion
Solvers that are currently vouched under the CoW DAO Bonding pool may decide to set up a reduced bonding pool according to [CIP-44](https://snapshot.box/#/s:cow.eth/proposal/0x1b6f1171633ec3d20c4370db37074aa1bd830486d4d0d6c26165915cc42d9412). Note that the interested solver team first needs to coordinate with the core team, that is currently managing the CoW DAO bonding pool, and the core team maintains the right to reject such a reduced pool setup.
```

File: docs/cow-protocol/reference/core/auctions/bonding_pools.md [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+
+The first step for setting up a reduced bonding pool is to deploy a Gnosis safe on Mainnet that has only one signer ([the CoW DAO Solver Payouts safe](0xA03be496e67Ec29bC62F01a428683D7F9c204930)). After this is done and has been confirmed by the CoW DAO team, the solver will deposit $50.000 in yield bearing stable coins or ETH and 500.000 COW tokens to the newly created safe.
+
+# How to join a bonding pool
+This is done by vouching for a solver's submission address and rewards address with the bonding pool address. This is done by calling the `Vouch` method on the [VouchRegister contract](https://etherscan.io/address/0xb422f2520b0b7fd86f7da61b32cc631a59ed7e8f) using the address that owns the bonding pool (the address that created the bonding pool).
```
Comment (harisang) [inline docs/cow-protocol/reference/core/auctions/bonding_pools.md]:
```suggestion
This is done by vouching for a solver's submission address and rewards address with the bonding pool address. This is done by calling the `Vouch` method on the VouchRegister contract using the address that created the bonding pool.
```

File: docs/cow-protocol/reference/core/auctions/bonding_pools.md [hunk 12]
```diff
@@ -0,0 +1,32 @@
...
+# How to join a bonding pool
+This is done by vouching for a solver's submission address and rewards address with the bonding pool address. This is done by calling the `Vouch` method on the [VouchRegister contract](https://etherscan.io/address/0xb422f2520b0b7fd86f7da61b32cc631a59ed7e8f) using the address that owns the bonding pool (the address that created the bonding pool). 
+
+Vouching contracts:
+- [Mainnet](https://etherscan.io/address/0xb422f2520b0b7FD86f7DA61b32Cc631A59ed7E8F)
+- [Gnosis Chain](https://gnosisscan.io/address/0xAAA4De096D02AE21729aA31D967E148D4e3Ae501)
+- [Arbitrum](https://arbiscan.io/address/0xAAA4De096D02AE21729aA31D967E148D4e3Ae501)
+- [Base](https://basescan.org/address/0xAAA4De096D02AE21729aA31D967E148D4e3Ae501)
+
+The `Vouch` method receives a list as an argument and can vouch for multiple submission addresses in a single transaction. The transaction must contain: the submission address for the solver, the bonding pool that is vouching for the solver, and the rewards address that the solver would like to use to receive their [rewards](https://docs.cow.fi/cow-protocol/reference/core/auctions/rewards).
```
Comment (harisang) [inline docs/cow-protocol/reference/core/auctions/bonding_pools.md]:
```suggestion
The `Vouch` method receives a list as an argument and can vouch for multiple submission addresses in a single transaction. The transaction must contain: the submission address for the solver, the bonding pool that is vouching for the solver, and the rewards address that the solver would like to use to receive their [rewards](/cow-protocol/reference/core/auctions/rewards).
```

**Comment by bram-vdberg** [comment]
I have read the CLA Document and I hereby sign the CLA

File: docs/cow-protocol/reference/core/auctions/bonding_pools.md (hunk 8 above)
Comment (bram-vdberg) [reply docs/cow-protocol/reference/core/auctions/bonding_pools.md]: Thanks, sorry about that. I keep forgetting this.

File: docs/cow-protocol/reference/core/auctions/bonding_pools.md [hunk 13]
```diff
@@ -0,0 +1,32 @@
...
+
+Once this safe has been confirmed by the CoW DAO team, the safe should be funded with $500,000 USD in yield bearing stable coins and 1,500,000 COW tokens. After this is done, the bonding pool can be used to vouch for solvers in the solver competition.
+
+# Setting up a reduced bonding pool
+Solvers that are currently vouched under the CoW DAO Bonding pool may decide to set up a reduced bonding pool according to [CIP-44](https://snapshot.box/#/s:cow.eth/proposal/0x1b6f1171633ec3d20c4370db37074aa1bd830486d4d0d6c26165915cc42d9412). Note that the interested solver team first needs to coordinate with the core team, that is currently managing the CoW DAO bonding pool, and the core team maintains the right to reject such a reduced pool setup.
+
+The first step for setting up a reduced bonding pool is to deploy a Gnosis safe on Mainnet that has only one signer ([the CoW DAO Solver Payouts safe](0xA03be496e67Ec29bC62F01a428683D7F9c204930)). After this is done and has been confirmed by the CoW DAO team, the solver will deposit $50,000 in yield bearing stable coins or ETH and 500,000 COW tokens to the newly created safe.
```
Comment (harisang) [inline docs/cow-protocol/reference/core/auctions/bonding_pools.md]:
```suggestion
The first step for setting up a reduced bonding pool is to deploy a Gnosis safe on Mainnet that has only one signer (the CoW DAO Solver Payouts safe, that is, eth:0xA03be496e67Ec29bC62F01a428683D7F9c204930). After this is done and has been confirmed by the CoW DAO team, the solver will deposit $50,000 in yield bearing stable coins or ETH and 500,000 COW tokens to the newly created safe.
```

**Pull request merged**